### DISCUSSION TITLE
Display specific settings

### DISCUSSION THREAD

**Discussion by JurTer** [rxhanson/Rectangle#563] [Ideas]
Hi,

I am well aware of this being an edge case, but I'm starting this discussion hoping I've simply overlooked an existing product feature.

As a big fan of the 'gaps between windows' setting I find myself opening the Rectangle settings multiple times a day, every time I connect/disconnect my external display.

When using the larger external display I enjoy having a gap between the multiple (well organized :) windows, but when only using my 13' MacBook display I cannot afford to waste any pixels.

It would be amazing if some of the settings could be display specific. 
If window is on screen with size x by x then use gap of x pixels and when window is on screen with size y by y then use gap of y pixels.

Thanks,
Jurjen

**Comment by rxhanson** [comment]
Makes sense to me. I see no reason to reject a good PR with this feature.
